demisto_sdk/commands/create_id_set/tests/update_id_set_test.py

```diff
@@ -70,6 +76,42 @@ def test_create_id_set_no_output(self, mocker):
         assert 'Widgets' in id_set.keys()
         assert 'Mappers' in id_set.keys()
 
+    def test_create_id_set_on_specific_pack(self, repo):
```

Comment:
1. Let's add given - when - then to all the tests, you can check for examples in content repo
2. Let's try to work on tests readability. Make sure it is clear what the flow is and what is being tested + the inner logic of each test